File: 1/C/1.c
#include <stdlib.h>


int* twoSum(int* nums, int numsSize, int target, int* returnSize){
    
    int i=0;
    int j=0;
    int sum=0;
    *returnSize=2;
    int *res= (int*)malloc(2*sizeof(int));
    
    for (i=0;i<numsSize;i++){
        for (j=i+1;j<numsSize;j++){
            
            sum=nums[i]+nums[j];
            if (target==sum){
                
                res[0]=i;
                res[1]=j;
                return res;
            }
        }
    }
    res[0]=-1;
    res[1]=-1;
    return res;
}

File: 1.Two_Sum/python/1.py
from typing import List


def twoSum(nums: List[int], target: int) -> List[int]:
    hash_map = {}
    for i in range (len(nums)):
        hash_map[nums[i]] = i

    for i in range (len(nums)):
        if (target - nums[i]) in hash_map and hash_map[target - nums[i]] != i:
            return (i, hash_map[target - nums[i]])